scripts/create-clientconfig.sh

```diff
@@ -53,20 +49,103 @@ namespace=""
 output_dir=""
 in_cluster_kubeconfig=""
 
-while true; do
+while test $# -gt 0; do
   case "$1" in
-    --src-context ) src_context="$2"; shift 2 ;;
```

Comment:
I don't want to pollute the review with discussion around removal of gnu-getopt, so I added https://github.com/k8ssandra/k8ssandra-operator/issues/141#issuecomment-924306583.

Reply:
dealt with there, I'll reintroduce gnu-getopt with a proper check.